### DISCUSSION TITLE
ARIA role="comment" for comments

### DISCUSSION THREAD

**Discussion by Malvoz** [community/community#14029] [Accessibility] [answered]
ARIA 1.3 (draft) defines [`role="comment"`](https://w3c.github.io/aria/#comment).

It may improve the experience for screen reader users in supporting UAs if comments on Github were given this role.

**Comment by benjiallen** [comment]
@Malvoz do you know if any assistive technology (AT) supports this role? It sounds like an interesting addition to ARIA but I don't think this would be something we would prioritize until AT has adopted support.

**Reply by aardrian** [reply]
Support in Chrome is behind flags. Firefox and Safari have support, but are limited on macOS by the platform APIs.

The `comment` role was meant to be used in document editing, such as a Google Doc or Microsoft Word. It is designed to work in conjunction with a related set of ARIA bits referred to as ARIA Annotations: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Annotations#comments

I am not sure most comments on GitHub would benefit from this and may end up making the experience too verbose.

**Reply by aardrian** [reply]
I meant to link this other discussion since this should be evaluated in conjunction with that one: https://github.com/github-community/community/discussions/14030

**Reply by benjiallen** [reply]
Thanks for the additional context @aardrian.

**Comment by cehfisher** [comment, marked as answer]
Hello @Malvoz 

Thank you for your feedback, and we genuinely appreciate your input. Our team has reviewed your request and we want to be transparent about our priorities. This particular request is not currently being actively worked on. This doesn’t imply that we’re not considering it in the future, but for now, it is not on our development roadmap.

We encourage you to check our [public roadmap](https://github.com/github/roadmap/projects/1) to get insights into the projects we are currently focusing on for the upcoming year. Please note that smaller fixes may not always be listed explicitly but can often be addressed when related work is in progress.

As a valuable GitHub user, your feedback in this Discussion space, accompanied by appropriate tags and examples, plays a crucial role in shaping GitHub’s development. While we can’t guarantee immediate action on every piece of feedback, rest assured that we are attentive to your suggestions and always striving to enhance the GitHub experience.

Once again, thank you for taking the time to share your thoughts with us, even if the particular request is not currently in active development. Your input is valuable and contributes to the continuous improvement of GitHub!
